modules/ROOT/pages/_partials/push/registering_device.adoc

```diff
@@ -0,0 +1,143 @@
+// For more information, see: https://redhat-documentation.github.io/modular-docs/
+
+[id='registering-device']
+= Registering device on {unifiedpush-service} Service
+
+To receive native push notifications from a Push Network (e.g. APNs, FCM), the mobile device is identified with a unique device-token, assigned by the actual Push Network. This device-token is passed, by the underlying Operating-System, to the mobile application. Details are different on each platform and not part of this document.
+
+Everytime when a mobile application launches it receives the above mentioned ```device-token```, via a _platform-specific_ method (or callback). Since the Push Network (e.g. APNs or FCM) _may_ assign a **new** token to a device, it is _recommended_ to _always_ (re)register the ```device-token``` with the _AeroGear UnifiedPush Server_.
```

Comment:
```suggestion
Every time a user launches a mobile app,  that app receives the  ```device-token```, from a _platform-specific_ method (or callback). 
Since the Push Network  may assign a new token to a device, {org-name} recommends that the app registers the ```device-token``` with the UnifiedPush Server each time.
```